packages/mc-scripts/src/config/create-postcss-config.ts

```diff
@@ -67,13 +67,7 @@ function createPostcssConfig({
        */
       require('postcss-custom-properties')({
         preserve: false,
-        importFrom: [
-          safeResolvePath(
-            '@commercetools-uikit/design-system',
-            'materials/custom-properties.css'
-          ),
```

Comment:
Removing this part as the `ThemeProvider` will now provide the custom CSS variables in the DOM so this is not needed anymore.